packages/select/src/components/ui/ChipMultiSelect/ChipMultiSelect.tsx

```diff
@@ -3,48 +3,48 @@ import * as React from "react";
 import { MultiSelect, MultiSelectProps } from "../MultiSelect";
 import { ChipRow, ChipRowItem } from "./ChipRow";
 
-export interface ChipMultiSelectValue extends ChipRowItem {}
-
-export interface ChipMultiSelectProps
-  extends Omit<MultiSelectProps, "value" | "onChange" | "isLoading">,
-    ValueAndOnValueChangeProps<Array<ChipMultiSelectValue>> {
+export interface ChipMultiSelectProps<TOption>
+  extends Omit<MultiSelectProps<TOption>, "value" | "onChange" | "isLoading">,
+    ValueAndOnValueChangeProps<ReadonlyArray<TOption>> {
```

Comment:
Maybe forcing the use of `ReadonlyArray` is bad. But that is what react-select uses.